backend/mailru/mailru_test.go

```diff
@@ -0,0 +1,23 @@
+// Test Mailru filesystem interface
+package mailru_test
+
+import (
+	"testing"
+
+	"github.com/rclone/rclone/backend/mailru"
+	"github.com/rclone/rclone/fstest"
+	"github.com/rclone/rclone/fstest/fstests"
+)
+
+// TestIntegration runs integration tests against the remote
+func TestIntegration(t *testing.T) {
+	remoteName := *fstest.RemoteName // from the -remote cli option
+	if remoteName == "" {
+		remoteName = "TestMailru:"
```

Comment:
Just specify `RemoteName: "TestMailru:"` and when the `-remote` flag is passed it will get overridden automatically.